### DISCUSSION TITLE
Today's Detections broken screen

### DISCUSSION THREAD

**Discussion by AndyZaa** [Nachtzuster/BirdNET-Pi#146] [Q&A]
I've enjoyed putting together my BirdNET Pi, but I have a persistent problem with my Today's Detections screen.

Most of the time I get a screen like this:
![Birdnet Today's Detections 01](https://github.com/user-attachments/assets/ee9cc001-03b4-40f3-b540-358eecfc407b)
where the audio content is apparently not there - if you hit the Play button, a wheel spins for an instant and nothing plays. Then that button is greyed out until you refresh.
I know the audio is there because I can see a normal screen with spectrum and audio for the first recording in the Overview screen.

Sometimes I get a fully functional screen like this:
![Birdnet Today's Detections 02](https://github.com/user-attachments/assets/abfd2d9b-e52b-4e33-82e0-6a3f01580a03)
where the full spectrum is shown and the audio plays properly.

I'm running on a Pi 4 Model B 4GB, and I'm fully updated through the Tools>Update menu. I've rebooted, cleared data, restarted core services. Nothing helps. 

Anyone seeing something similar? You should be able to see my BirdNET at [http://zbirdnet.duckdns.org/](http://zbirdnet.duckdns.org/), and my station is [https://app.birdweather.com/stations/5280](https://app.birdweather.com/stations/5280).

I'm hoping someone can help because I can't think of what else to do but a clean install. Thanks!

**Comment by alexbelgium** [comment]
Just tried and it shows fully fine in my browser (edge). I wonder if that is not because you need to wait for the whole page to load before loading medias. What is the type of audio format you have? mp3?

Ps: very nice quality of recording, what is your mic system ? Apart from the extremely noisy crickets the spectrogram looks great. Btw the spectrograms looks like there is some distortion likely because you have set your audio to 1 channel in options. I had the same and selecting 2 channels (from a mono mic) led to normal spectrograms. Never thought too much about it though.

![image](https://github.com/user-attachments/assets/a6e68c58-9017-4aee-9b7f-c34ae6752230)

**Reply by AndyZaa** [reply]
Thanks, Alexandre. For some reason, this morning my Today's Detections is working as it should. Mysterious, as I haven't changed anything. If it reverts to the previous behavior, I will try in other browsers - I know I tried that in the past without resolution but I'll do it in a more structured manner. I do know it was behaving the same way in my iOS devices.

I'm using a "Movo WMX-LAV Lavalier Microphone" with the widely recommended "UGREEN USB to Audio Jack Sound Card Adapter with Dual TRS 3-Pole Headphone and Microphone USB to Aux 3.5mm External Audio Converter". I experimented with the naked mic, dead cat, and foam cover - currently using a foam cover. When I first put the Pi together and before I received the mic and sound card, I was using the RTSP from my Amcrest surveillance camera, and that was actually quite good.

Thanks for the tip on the audio channels. I did have it set to 1 and just changed it back to the default 2. I am saving as mp3.

Recently I used Tools > Web Terminal, "alsamixer -m" to set both the Master and Capture adjustments to max (all the way in the red) because both my input and output seemed low. I haven't noticed any distortion, and the audio levels have improved.

**Reply by alexbelgium** [reply]
Thanks for the info! Looks like indeed a very nice setup
